src/v2/providers/pubsub.ts

```diff
@@ -4,20 +4,25 @@ import { CloudEvent, CloudFunction } from '../core';
 import * as options from '../options';
 
 /**
- * A PubSub Topic is:
+ * Google Cloud's Pub/Sub is a globally distributed message bus that automatically scales as you need it.
```

Comment:
I think this should be just `Google Cloud Pub/Sub`

Reply:
+1